lib/github_service/commands/cross_repo_test.rb

```diff
@@ -134,18 +137,40 @@ def parse_value(value)
         @value = value
 
         @test_repos, @repos = value.split(/\s+including\s+/)
-                                   .map { |repo_list| repo_list.split(",") }
+                                   .map { |repo_list| repo_list.split(",").map(&:strip) }
```

Comment:
This was causing issues with multiple test_repos like `repo1, /providers` because the latter would be `" /providers"` and the `start_with?("/")` was failing, seemed better to just strip all leading/trailing whitespace anyway